src/components/policy/src/policy/src/policy_manager_impl.cc

```diff
@@ -147,10 +147,20 @@ bool PolicyManagerImpl::LoadPT(const std::string& file,
     listener_->OnCertificateUpdated(*(pt_update->policy_table.module_config.certificate));
   }
 
-
   // Check permissions for applications, send notifications
   CheckPermissionsChanges(pt_update, policy_table_snapshot);
 
+#ifdef SDL_REMOTE_CONTROL
+  // Check country_consent
+  bool has_changed = access_remote_->CheckPTUUpdatesChange(pt_update,
+                                        policy_table_snapshot);
+  access_remote_->Init();
```

Comment:
I wanted avoid use this call. So I replaced country_consent_ filed on country_consent() method. But it isn't enough I see now. :(